sysv_argv.dart fails on Omni image

I've not investigated this, as I havent seen it locally:

```python
_______________ TestCodeGenerator.test_generated[sys_argv-dart] ________________
Traceback (most recent call last):
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/runner.py", line 353, in from_call
    result: TResult | None = func()
                             ^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/runner.py", line 245, in <lambda>
    lambda: runtest_hook(item=item, **kwds),
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_hooks.py", line 512, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
    raise exception
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/logging.py", line 850, in pytest_runtest_call
    yield
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 53, in run_old_style_hookwrapper
    return result.get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_result.py", line 103, in get_result
    raise exc.with_traceback(tb)
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 38, in run_old_style_hookwrapper
    res = yield
          ^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/capture.py", line 900, in pytest_runtest_call
    return (yield)
            ^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 139, in _multicall
    teardown.throw(exception)
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/skipping.py", line 268, in pytest_runtest_call
    return (yield)
            ^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 121, in _multicall
    res = hook_impl.function(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/runner.py", line 179, in pytest_runtest_call
    item.runtest()
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/python.py", line 1720, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_hooks.py", line 512, in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_manager.py", line 120, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 167, in _multicall
    raise exception
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/pluggy/_callers.py", line 121, in _multicall
    res = hook_impl.function(*args)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/.tox/py312-full/lib/python3.12/site-packages/_pytest/python.py", line 166, in pytest_pyfunc_call
    result = testfunction(**testargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/__w/py2many/py2many/tests/test_cli.py", line 316, in test_generated
    expected_exit_code == proc.returncode
AssertionError: Execution of sys_argv.dart failed:
b''b'Unhandled exception:\n\'file:///__w/py2many/py2many/tests/build/sys_argv.dart\': Failed assertion: line 12 pos 12: \'cmd.contains("sys_argv")\': is not true.\n#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)\n#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)\n#2      main (file:///__w/py2many/py2many/tests/build/sys_argv.dart:12:12)\n#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)\n#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)\n'
----------------------------- Captured stdout call -----------------------------
Wrapping unimplemented in comments
/__w/py2many/py2many/tests/cases/sys_argv.py ... /__w/py2many/py2many/tests/build/sys_argv.dart
expected = generated
Compiling ['dart', 'compile', 'exe', '/__w/py2many/py2many/tests/build/sys_argv.dart'] ...
Generated: /__w/py2many/py2many/tests/build/sys_argv.exe
Invoking ['dart', '--enable-asserts', '/__w/py2many/py2many/tests/build/sys_argv.dart', 'arg1'] ...
----------------------------- Captured stderr call -----------------------------
Writing to: /__w/py2many/py2many/tests/build
```

--- a/README.md
+++ b/README.md
@@ -132,7 +132,7 @@ clang tests/expected/fib.cpp
 ./scripts/rust-runner.sh run tests/expected/fib.rs
 
 # Run D
-dmd -run tests/cases/fib.d
+ldmd2 -run tests/cases/fib.d
 ```
 
 **Language-Specific Tools:**

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -63,8 +63,6 @@ RUN apk add --no-cache \
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-RUN ln -sf /usr/bin/ldmd2 /usr/local/bin/dmd
-
 RUN should_install() { case "${FOCUS}" in ""|all|omni|"${1}") return 0 ;; *) return 1 ;; esac; }; \
     echo "Building py2many CI image for focus: ${FOCUS:-all}"
 

--- a/docker/Dockerfile.omni
+++ b/docker/Dockerfile.omni
@@ -63,8 +63,6 @@ RUN apt-get update \
       zip \
     && rm -rf /var/lib/apt/lists/*
 
-RUN ln -sf /usr/bin/ldmd2 /usr/local/bin/dmd
-
 RUN arch="$(dpkg --print-architecture)" \
     && case "${arch}" in \
       amd64) go_arch="amd64"; zig_arch="x86_64"; julia_path_arch="x64"; julia_tar_arch="x86_64"; nim_arch="x64" ;; \

--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -47,7 +47,7 @@ COMPILERS = {
     + _conan_include_args()
     + (["-stdlib=libc++"] if CXX.startswith("clang++") else [])
     + (["-o", "{exe}", "{filename}"] if sys.platform == "win32" else []),
-    "dlang": ["dmd"],
+    "dlang": ["ldmd2"],
     "dart": ["dart", "compile", "exe"],
     "go": ["go", "build"],
     "nim": ["nim", "compile", "--nimcache:."],
@@ -65,7 +65,7 @@ COMPILERS = {
 
 INVOKER = {
     "dart": ["dart", "--enable-asserts"],
-    "dlang": ["dmd", "-run"],
+    "dlang": ["ldmd2", "-run"],
     "go": ["go", "run"],
     "julia": ["julia", "--compiled-modules=yes"],
     "python": [sys.executable],

